package.json

```diff
@@ -26,8 +26,8 @@
 		"ci:build": "fluid-build --task ci:build",
 		"ci:test:jest": "npm run test:jest:report",
 		"ci:test:jest:coverage": "c8 --no-clean npm run test:jest:report",
-		"ci:test:mocha": "npm run test:mocha:report",
-		"ci:test:mocha:coverage": "c8 --no-clean npm run test:mocha:report",
+		"ci:test:mocha": "npm run test:mocha",
+		"ci:test:mocha:coverage": "c8 --no-clean npm run test:mocha -- --timeout 4s",
```

Comment:
Is this right? Should this not be calling `test:mocha:coverage`?

Reply:
`c8` executable in front will inject coverage collection to all child process.   It is faster than calling `c8` on each individual command the last I checked.